### DISCUSSION TITLE
Git commit graph not updating

### DISCUSSION THREAD

**Discussion by ChandanBose666** [community/community#40383] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I have been committing to a certain repository almost on a daily basis but the git graph on the profile page doesn't seem to update accordingly. Can someone help? I have attached screenshots of the same.
![git graph screenshot](https://user-images.githubusercontent.com/94798980/204626575-914aeb69-5b08-4d67-a867-d9908d44dd70.png)
![commit ss](https://user-images.githubusercontent.com/94798980/204626706-764a8f3e-25b3-4143-bdf5-b7db1e0b402e.png)

**Comment by airtower-luna** [comment]
The screenshot shows that the commits aren't recognized as yours (otherwise it'd show your profile picture). Check your `user.email` setting. If the address is the one you want to use, you need to add it to your account. If not you need to fix the setting, see: https://docs.github.com/en/github/setting-up-and-managing-your-github-user-account/managing-email-preferences/setting-your-commit-email-address

**Reply by ChandanBose666** [reply]
Solved by changing the config file. Thanks though
